packages/@okta/vuepress-site/docs/api/resources/oauth-clients/index.md

```diff
@@ -276,7 +276,7 @@ Lists all clients that match a search filter on `client_name`
 
 
 Parameter | Description                                                                                | ParamType | DataType | Required | Default | Maximum |
```

Comment:
Why do the other tables start with a | but this one does not?

Reply:
Missed this one... thanks!